src/cli/README.md

```diff
@@ -367,27 +367,108 @@ Done
 Done
 ```
 
+### br state
```

Comment:
Is it worth separating these into a separate `README_BORDER_ROUTER.md` doc?

Reply:
Good idea. the `diff` will be larger but I think it is ok (since it's already checked/reviewed).

Added `README_BR.md` (used short form `BR` to match the command name) which now conatins all the `br` sub-commands.